api/docs/release.dox

```diff
@@ -139,6 +139,8 @@ Further non-compatibility-affecting changes include:
    -core_serial) and control the schedule
```

Comment:
> Fixes the existing behavior where the post-syscall timestamp contains the pre-syscall time.

Which post-syscall timestamp do you mean? We now have a new post-syscall timestamp, so the issue in your statement is fixed automatically?

Reply:
I mean the trace buffer starting at the post-syscall point: the reason #6289 was filed in the first place.  I made it "post-syscall-buffer timestamp" in the description.